app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int = 0,
+                 clean_mark: int = 0,
+                 brand: str = 0) -> None:
```

Comment:
The `brand` attribute is expected to be a string, but it is initialized with an integer `0`. Consider changing the default value to an empty string `''` or a more appropriate default string value.